test/quantization/eager/test_quantize_eager_ptq.py

```diff
@@ -199,6 +199,61 @@ def test_linear(self):
             (16, 5)
         )
 
+    def test_int16_reference_module(self):
+
+        class RefM(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = nn.ConvTranspose2d(1, 1, 1)
+                self.conv.weight = torch.nn.Parameter(torch.randn_like(self.conv.weight), requires_grad=False)
+                self.conv.bias = torch.nn.Parameter(torch.randn_like(self.conv.bias), requires_grad=False)
+                self.quant1 = QuantStub()
+                self.dequant1 = DeQuantStub()
+                self.quant2 = QuantStub()
+                self.dequant2 = DeQuantStub()
+
+            def forward(self, x):
+                x = self.quant1(x)
+                x = self.dequant1(x)
+                x = self.conv(x)
+                x = self.quant2(x)
+                x = self.dequant2(x)
+                return x
+
+
+        input_size = (16, 1, 10, 10)
+        data = torch.randn(*input_size, dtype=torch.float)
+
+        original_ref_m = RefM()
+        torch.quantization.engine = "qnnpack"
```

Comment:
note: not all test environments will support qnnpack.  You can check for this with `"qnnpack" in torch.testing._internal.common_quantized.supported_qengines`, and skip the test if this is false